stable/gcloud-sqlproxy/templates/serviceaccount.yaml

```
@@ -1,9 +1,13 @@
-{{- if .Values.rbac.create }}
+{{- if .Values.serviceAccount.create }}
 apiVersion: v1
 kind: ServiceAccount
 metadata:
   namespace: {{ .Values.namespace }}
   labels:
     {{- include "gcloud-sqlproxy.labels" . | nindent 4 }}
+{{- with .Values.serviceAccount.annotations }}
+  annotations:
+{{ toYaml . | indent 4 }}
+{{- end }}
   name: {{ include "gcloud-sqlproxy.fullname" . }}
```

```suggestion
  name: {{ include "gcloud-sqlproxy.serviceAccountName" . }}
```

The above will require to also add in the helpers file:

```
{{/*
Create the name of the service account to use
*/}}
{{- define "gcloud-sqlproxy.serviceAccountName" -}}
{{- if .Values.serviceAccount.create -}}
    {{ default (include "gcloud-sqlproxy.fullname" .) .Values.serviceAccount.name }}
{{- else -}}
    {{ default "default" .Values.serviceAccount.name }}
{{- end -}}
{{- end -}}
```

---

Please also adjust the sa name in rbac templates as `name: {{ include "gcloud-sqlproxy.serviceAccountName" . }}`